src/base/bittorrent/session.cpp

```diff
@@ -4388,6 +4391,13 @@ void Session::createTorrentHandle(const lt::torrent_handle &nativeHandle)
     // Torrent could have error just after adding to libtorrent
     if (torrent->hasError())
         LogMsg(tr("Torrent errored. Torrent: \"%1\". Error: %2.").arg(torrent->name(), torrent->error()), Log::WARNING);
+#if (LIBTORRENT_VERSION_NUM < 10200)
```

Comment:
Please add comments about why we need it above this line and add an empty blank line to separate it with others.